app/helpers/admin/users_helper.rb

```diff
@@ -0,0 +1,25 @@
+module Admin
+  module UsersHelper
+    def format_last_activity_timestamp(timestamp)
+      if timestamp.today?
+        "Today, #{timestamp.strftime('%d %b')}"
+      elsif timestamp.yesterday?
+        "Yesterday, #{timestamp.strftime('%d %b')}"
+      else
+        timestamp.strftime("%d %b, %Y")
+      end
+    end
+
+    def cascading_high_level_roles(user)
+      if user.super_admin?
+        "Super Admin"
+      elsif user.admin?
+        "Admin"
+      elsif user.trusted?
+        "Tag moderator"
+      else
+        "Regular member"
+      end
```

Comment:
I'm not sure why I thought that the "Regular member" badge was a thing... I must be losing it. 😪 I'll remove it.

Reply:
I think with this change we've introduced a tiny issue where when it's empty, we show an empty little circle instead of the role:

![small grey dot](https://user-images.githubusercontent.com/20773163/162176486-beea8057-ca2e-4ea4-a045-6c3623902e74.png)

I'll add correcting this as a task on the overall issue as I don't think it's been captured yet - should be a quick fix I'm sure!